backend/TaskTree/Views/Projects1/Create.cshtml

```diff
@@ -0,0 +1,53 @@
+﻿@model TaskTree.Models.Project
```

Comment:
there shouldn't be any views, this is an api only application so we return the response objects from the controller and they are rendered as json.